kt/godot-library/src/main/kotlin/godot/tests.kt

```diff
@@ -11,24 +11,24 @@ open class Object(isRef: Boolean = false) : KtObject(isRef) {
     constructor() : this(false)
 
     override fun __new(): VoidPtr {
-        return TransferContext.invokeConstructor("Object")
+        return TransferContext.invokeConstructor(0)
```

Comment:
Couldn't we use `const val`'s for all these constant integers? would improove readability in my opinion and  the compiler should optimize these constants anyways if they can't change.
even though we'll generate these classes later on, once one needs to debug them, having variables in the source code rather than plain numbers might be beneficial?

Reply:
I agree, let me change this, so that I remember for later entry generation.